src/platform/ESP32/NetworkCommissioningDriver.cpp

```diff
@@ -38,6 +38,25 @@ constexpr char kWiFiCredentialsKeyName[] = "wifi-pass";
 static uint8_t WiFiSSIDStr[DeviceLayer::Internal::kMaxWiFiSSIDLength];
 } // namespace
 
+CHIP_ERROR GetConfiguredNetwork(Network & network)
+{
+    wifi_ap_record_t ap_info;
+    esp_err_t err;
+    err = esp_wifi_sta_get_ap_info(&ap_info);
```

Comment:
Does this fail if the network is not actually joined?  @wqx6

Reply:
Yes, this function will return ESP_ERR_WIFI_CONN when the station interface is not initialized and ESP_ERR_WIFI_NOT_CONNECT when the station is in disconnect status.